src/routes/userRoutes.js

```diff
@@ -5,14 +5,16 @@ const {
   registerUser,
   loginUser,
   getLoggedInUser,
-  updateUser
+  updateUser,
+  searchUsers 
 } = require("../controllers/userController");
 const { protect } = require("../middleware/authMiddleware");
 
 router.post("/signup", registerUser); // should be sign up /api/signup
 router.post("/login", loginUser); // /api/login
 router.get("/me", protect, getLoggedInUser); //api/user/{id}
 router.put("/:id", updateUser); 
+router.get("/search", searchUsers);
```

Comment:
Looks like /search endpoint should be protected and accessible only for authorize users